src/views/settings/Applet/AppletHost/AppletHostCreateUpdate.vue

```diff
@@ -78,8 +75,6 @@ export default {
     }
   },
   async mounted() {
-    const platform = await this.$axios.get('/api/v1/assets/platforms/RemoteAppHost/')
-    this.$route.query.platform = platform['id']
     this.loading = false
   },
   methods: {}
```

Comment:
The code was changed to use camelCase instead of underscore notation, which is consistent with many popular JavaScript styles. Additionally, the second "addFields" block added to both components should be merged into one to reduce redundancy and improve the efficiency of adding fields later on.